src/app/copy/copy-config-base.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * A config containing properties for copy components
+ */
+export class CopyConfigBase {
```

Comment:
I believe we can remove this config class and move the typedoc comments to CopyBase?